src/tests/Microsoft.PowerFx.Interpreter.Tests/MutationScripts/AndOr.txt

```diff
@@ -0,0 +1,72 @@
+﻿// Case to test how shortcut verification work along with behavior functions
```

Comment:
All of the `And` tests are short-circuiting (false && <something>), while all of the `Or` tests are not (false Or <something>). We should have tests that short-circuit the Or cases as well.